test/services/sync/repo/user.repo.integration.test.js

```diff
@@ -187,4 +188,79 @@ describe('user repo', () => {
 			expect(res[1]._id.toString()).to.be.equal(createdUsers[1]._id.toString());
 		});
 	});
+
+	describe('createOrUpdateImportUser', () => {
+		it('should create import user', async () => {
+			const school = await testObjects.createTestSchool();
+			const system = await testObjects.createTestSystem();
+			const user = {
+				firstName: 'New fname',
+				lastName: 'New lname',
+				email: 'new email',
+				ldapDn: 'new ldapdn',
+				ldapId: `ldapId${Math.random()}`,
+				roles: ['role1'],
+			};
+			await UserRepo.createOrUpdateImportUser(school._id, system._id, user.ldapId, user);
+			const res = await importUserModel.findOne({ schoolId: school._id, ldapId: user.ldapId }).lean().exec();
+			const attributesToCompare = ['firstName', 'lastName', 'email', 'ldapDn'];
+			attributesToCompare.forEach((attr) => expect(res[attr]).to.be.equal(user[attr]));
+			expect(res.roles).to.eql(user.roles);
+		});
+
+		it('should update import user', async () => {
+			const testSystem = await testObjects.createTestSystem();
+			const school = await testObjects.createTestSchool();
+			const importUser = await testObjects.createTestImportUser({ schoolId: school._id, system: testSystem._id });
+
+			const user = {
+				firstName: 'New fname',
+				lastName: 'New lname',
+				email: 'new email',
+				ldapDn: 'new ldapdn',
+				ldapId: importUser.ldapId,
+				roles: ['role1'],
+			};
+			await UserRepo.createOrUpdateImportUser(school._id, testSystem._id, user.ldapId, user);
+			const res = await importUserModel.find({ schoolId: school._id, ldapId: user.ldapId }).lean().exec();
+			expect(res.length).to.equal(1);
+			const attributesToCompare = ['firstName', 'lastName', 'email', 'ldapDn'];
+			attributesToCompare.forEach((attr) => expect(res[0][attr]).to.be.equal(user[attr]));
+			expect(res[0].roles).to.eql(user.roles);
+			expect(res[0].system.toString()).to.equal(testSystem._id.toString());
+		});
+	});
+
+	describe('addClassToImportUsers', () => {
+		it('should create class to import users', async () => {
+			const testSystem = await testObjects.createTestSystem();
+			const school = await testObjects.createTestSchool();
+			const importUser = await testObjects.createTestImportUser({ schoolId: school._id, system: testSystem._id });
+
+			const userLdapDns = [importUser.ldapDn];
+			const className = '1a';
+			await UserRepo.addClassToImportUsers(school._id, className, userLdapDns);
```

Comment:
add test: when adding the same class name twice, the name should appear only one in classes